### ISSUE TITLE
Sphinx Documentation

### ISSUE THREAD

**Issue by mraunak** [ModECI/MDF#88]
**Overview**
Sphinx is document generator that converts text files, markdown files to HTML/PDF etc. formats

restructuredText, Markdown -> Sphinx -> HTML

- cd docs
- pip install -r requirements.txt
- Output Format:  html file

- Documentation for mdf.py, scheduler.py, standard_functions.py, onnx_functions.py, interfaces.pytorch.exporter.py
- Included readme files

**Features**

Google style Python docstrings is used for documenting python files. It requires sphinx napoleon extension
Napoleon is a pre-processor that parses Google style docstrings and converts them to reStructuredText before Sphinx attempts to parse them.

html theme : "sphinx read the doc theme"

file extensions supported currently:

".rst": "restructuredtext",
".txt": "restructuredtext",
".md": "markdown"

Intersphinx_mapping: Create automatic links to the documentation of modules and objects in other standard documents such as numpy, python

 'python': ('https://docs.python.org/', None)
 'numpy': ('https://numpy.org/doc/stable', None)

**Working On**
- Sphinx Contrib Versioning-build Sphinx docs for all versions of the project 
- Working on incorporating all readme files from different modules
- Automatic generation of links to map standalone repo of scheduler
- Graphviz in sphinx
                       
**Reference**

https://www.sphinx-doc.org/en/master/
https://sphinxcontrib-versioning.readthedocs.io/en/latest/tutorial.html
